source/usage-examples/convenient-transactions.txt

```diff
@@ -0,0 +1,162 @@
+.. _node-usage-convenient-txn:
+
+================================
+Perform a Convenient Transaction
+================================
+
+The {+driver-short+} provides two APIs to perform transactions, the Core
+API and the **Convenient Transaction API**. By using the Convenient Transaction
+API, you can perform a transaction without being responsible for
+ensuring the transaction was committed or aborted. This API
+automatically incorporates error handling logic for certain error types,
+such as ``TransientTransactionError`` and ``UnknownTransactionCommitResult``.
+
+This usage example provides sample code that uses the Convenient
+Transaction API. To learn more about the Core API and view an example,
+see the :ref:`nodejs-transactions` guide.
+
+.. tip::
+   
+   To learn more about the ``TransientTransactionError`` error, see the
+   :manual:`TransientTransactionError description
+   </core/transactions-in-applications/#transienttransactionerror>` on
+   the Driver API page in the Server manual.
+
+   To learn more about the ``UnknownTransactionCommitResult`` error, see the
+   :manual:`UnknownTransactionCommitResult description
+   </core/transactions-in-applications/#unknowntransactioncommitresult>` on
+   the Driver API page in the Server manual.
+
+The {+driver-short+} provides the following helper methods to perform a
+transaction by using the Convenient Transaction API:
+
+- ``withSession()``: Runs a given operation within an implicitly created
+  session. You do not have to handle the creation or ending of the
+  session.
```

Comment:
```suggestion
- ``withSession()``: Runs a given function passing it a started session. Regardless of the function's outcome, the session will automatically be ended and related resources cleaned up.
```

An "implicit" session is specific terminology that refers to sessions created by the driver for every operation if a user didn't provide one. In this case, the session is still "explicit" as the user still has to pass it to each operation.